wireless_watcher/nodes/watcher_node

```diff
@@ -37,6 +37,7 @@ class Connection(wireless_msgs.msg.Connection):
           args['link_quality_raw'] = fields['Link Quality']
           num, den = re.split('/', args['link_quality_raw'])
           args['link_quality'] = float(num) / float(den)
+          args['frequency'] = trim(fields['Frequency'], float)
```

Comment:
Maybe this line should not be in the try block but instead with the other args which use fields.get() to handle missing data from iwconfig. (The reason for keeping the try block is that 'link_quality' depends on data from 'link_quality_raw'.)